source/deploy/mobile-hpns.rst

```diff
@@ -49,7 +49,7 @@ To use the Mattermost TPNS, go to **System Console > Environment > Push Notifica
 See our `Testing Push Notifications <https://docs.mattermost.com/deploy/mobile-testing-notifications.html>`__ documentation to learn more about testing mobile push notifications.
 
 .. note::
-  - The TPNS only works with the pre-built mobile apps Mattermost deploys through the Apple App Store and Google Play Store. If you have built your own mobile apps, you must also `host your own Mattermost push proxy service <#host-your-own-push-proxy-service>`_.  
+  - The TPNS only works with the pre-built mobile apps that Mattermost deploys through the Apple App Store and Google Play Store. If you have built your own mobile apps, you must also `host your own Mattermost push proxy service <#host-your-own-push-proxy-service>`_.  
```

Comment:
> The TPNS only works with the pre-built mobile apps that Mattermost deploys through the Apple App Store and Google Play Store

also aplies to HPNS

Reply:
Confirmed that this note appears under the HPNS section as well.